tests/test_css_files.py

```
@@ -0,0 +1,35 @@
+    # Strip comments
+    clean_css = re.sub(r'/\*[\s\S]*?\*/', '', content)
+
+    # Check brace balance
+    open_braces = clean_css.count('{')
+    close_braces = clean_css.count('}')
+    assert open_braces == close_braces, f"Unbalanced braces in CSS file: {os.path.basename(filepath)}"
+
+def test_main_css_theme_variables():
+    main_css_path = os.path.join(ROOT_DIR, 'css', 'main.css')
+    assert os.path.exists(main_css_path)
+    with open(main_css_path, 'r', encoding='utf-8') as f:
```

Ruff (0.16.2)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)